src/extractors/custom/bookwalker.jp/index.js

```diff
@@ -0,0 +1,32 @@
+export const BookwalkerJpExtractor = {
+  domain: 'bookwalker.jp',
+
+  title: {
+    selectors: ['h1.main-heading'],
+  },
+
+  author: null,
+
+  date_published: null,
```

Comment:
I've noticed that the last item in the "work information" section is always the publishing date. Something like: `.work-info .work-detail:first-of-type .work-detail-contents:last-of-type`

Reply:
`.work-info .work-detail:first-of-type .work-detail-contents:last-of-type` is the publishing date of this novel, not the publishing date of this article.
I did not use `.date_published` selector because I thought it should set the publishing date of the article.

Reply:
Let's proceed with this change, as discussed in https://github.com/postlight/mercury-parser/pull/374#issuecomment-482359833